hack/install-kubebuilder.sh

```diff
@@ -2,6 +2,6 @@
 
 os=$(go env GOOS)
 arch=$(go env GOARCH)
-curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.1/kubebuilder_2.3.1_${os}_${arch}.tar.gz | tar -xz -C /tmp/
-sudo mv /tmp/kubebuilder_2.3.1_${os}_${arch} /usr/local/kubebuilder
-export PATH=$PATH:/usr/local/kubebuilder/bin
+version=3.9.0
+sudo curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${version}/kubebuilder_${os}_${arch} -o /usr/local/bin/kubebuilder
```

Comment:
have you verified that there are no breaking changes between these versions that will affect the operator?

Reply:
yes

Reply:
There seems to be lots of breaking changes in moving from 2.x.x to 3.x.x [here](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v3.0.0). Can you put in the description of the PR how you tested / confirmed that this will work as expected?